glibc 2.10 error: invalid conversion from 'const char*' to 'char*'
ref: http://lists.opensuse.org/opensuse-packaging/2009-06/msg00001.html
--- scim-1.4.7/src/ltdl.cpp.orig	Mon Jun 22 10:28:27 2009
+++ scim-1.4.7/src/ltdl.cpp	Mon Jun 22 10:25:11 2009
@@ -3360,7 +3360,7 @@
 {
   lt_dlhandle	handle		= 0;
   char *	tmp		= 0;
-  char *	ext		= 0;
+  const char *	ext		= 0;
   size_t	len;
   int		errors		= 0;
 
